### PR TITLE
Create 2024-01-26-governance-update.md

### PR DESCRIPTION
**Pull request by malvikasharan**
- announce the gov committee 
- update from 6 months of work

### CODE REVIEW

File: _posts/2024-01-26-governance-update.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+title: Announcing the OLS Governance and Their Work Since June 2023
+authors: 
+- malvikasharan
+- yochannah
+image: /images/2023-01-gov-post-almani-qhizq_V876M-unsplash.png
+---
+
+At OLS, we are thrilled to share a significant milestone: the establishment of the OLS Governance Committee. In this post, we highlight their work with us over the past six months.
+
+## Evolution of OLS
```
Comment (malvikasharan) [inline _posts/2024-01-26-governance-update.md:12]:
```suggestion
# Governance Focus of OLS in 2023
```

File: governance.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ layout: page
 title: Governance Committee
 image: /images/about.jpg
 photos:
-  name: Bérénice Batut
-  license: CC BY-SA 4.0
-  url: https://flic.kr/p/2gHNtNq
+  name: Benjamin Combs
+  license: Unsplash
+  url: images/2024-02-governance-page-benjamin-combs-wuU_SSxDeS0-unsplash.jpg
```
Comment (malvikasharan) [inline governance.md:8]:
```suggestion
  url: https://unsplash.com/photos/person-showing-green-plant-wuU_SSxDeS0
```

File: governance.md [hunk 3]
```diff
@@ -3,9 +3,9 @@ layout: page
 title: Governance Committee
 image: /images/about.jpg
```
Comment (malvikasharan) [inline governance.md:4]:
```suggestion
image: /images/2024-02-governance-page-benjamin-combs-wuU_SSxDeS0-unsplash.jpg
```

**Review by malvikasharan — COMMENTED** [review]
Adding image link

File: _posts/2024-01-26-governance-update.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ In June 2023, the OLS Board invited 10 community leaders from OLS and the open s
 
 ## Meet the Committee Members
 
-Learn more about the Governance Committee and its members on our [governance page](https://openlifesci.org/governance.html). Individual committee members are listed below:
+Learn more about the Governance Committee and its members on our [governance page]({{ site.github.owner_url }}/governance). Individual committee members are listed below:
```
Comment (bebatut) [inline _posts/2024-01-26-governance-update.md:22]:
```suggestion
Learn more about the Governance Committee and its members on our [governance page]({% link governance.md %}). Individual committee members are listed below:
```

File: _posts/2024-01-26-governance-update.md [hunk 5]
```diff
@@ -60,4 +60,4 @@ As we enter the second phase, we are excited to leverage the Committee's advisor
 
 The journey continues, and we are very grateful to our Committee members for their generosity and time in OLS.
 
-You can follow updates on our [GitHub repository for governance](https://github.com/open-life-science/ols-governance). We look forward to keep sharing ongoing  discussions and developments with the broader OLS community!
+You can follow updates on our [GitHub repository for governance](https://github.com/open-life-science/ols-governance). We look forward to sharing ongoing  discussions and developments with the broader OLS community!
```
Comment (bebatut) [inline _posts/2024-01-26-governance-update.md:63]:
```suggestion
You can follow updates on our [GitHub repository for governance]({{ site.github.owner_url }}/ols-governance). We look forward to sharing ongoing  discussions and developments with the broader OLS community!
```

File: governance.md [hunk 6]
```diff
@@ -14,11 +14,13 @@ photos:
 # Governance Committee
 The OLS Governance Committee is a dedicated group of individuals appointed to provide strategic oversight and guidance in shaping the governance framework of OLS. This committee provides advisory support in decision-making, transparency, and sustainability within OLS.
 
+For governance-related documentation and updates, please visit our [public GitHub repository](https://github.com/open-life-science/ols-governance).
```
Comment (NPDebs) [inline governance.md:17]:
```suggestion
For governance-related documentation and updates, please visit our [public GitHub repository]({{ site.github.owner_url }}/ols-governance).
```

**Review by NPDebs — APPROVED** [review]
This looks near perfect, Malvika. 😍
I only made some formatting suggestions, especially to the names that needed more visibility.

Happy to merge afterwards. 🎉

**Comment by NPDebs** [comment]
Looks great. Thanks, Malvika! 🤍

**Comment by malvikasharan** [comment]
Thank you so much @NPDebs. Really appreciate the thorough review and improvements. Thank you for making it live. 💐

@bebatut I got the variables wrong again. 😅 Thank you for correcting.

**Pull request merged**